cfme/cloud/security_group.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+import attr
+
+from cfme.base.ui import BaseLoggedInPage
+from cfme.exceptions import ItemNotFound, SecurityGroupsNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigate_to, navigator
+from cfme.utils.wait import wait_for
+from navmazing import NavigateToAttribute, NavigateToSibling
+from widgetastic_manageiq import (
+    Accordion, BaseEntitiesView, BaseListEntity, BaseQuadIconEntity, BaseTileIconEntity,
+    BootstrapSelect, BreadCrumb, ItemsToolBarViewSelector, ManageIQTree, NonJSBaseEntity,
+    ParametrizedView, SummaryTable, Text, TextInput)
+from widgetastic_patternfly import Button, Dropdown, FlashMessages
+from widgetastic.widget import Select, View
+
+
+class SecurityGroupToolbar(View):
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    download = Dropdown('Download')
+
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class SecurityGroupDetailsToolbar(View):
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    download = Button(title='Download summary in PDF format')
+
+
+class SecurityGroupDetailsAccordion(View):
+    @View.nested
+    class properties(Accordion):  # noqa
+        tree = ManageIQTree()
+
+    @View.nested
+    class relationships(Accordion):  # noqa
+        tree = ManageIQTree()
+
+
+class SecurityGroupQuadIconEntity(BaseQuadIconEntity):
```

Comment:
Since you're using `BaseEntitiesView` in your All view, and these classes don't implement anything specific, they're unnecessary, along with the NonJSSecurityGroupEntity class - which isn't used anywhere.